ReactiveXComponentTest/IntegrationTests/RabbitMqCommunicationTest.cs

```diff
@@ -113,9 +116,9 @@ public void SendEvent_GivenAStateMachineAMessageAndAVisibility_ShouldSendMessage
                 throw new Exception("Message not received");
```

Comment:
You should avoid throwing exceptions is tests. Here I'd rather add a check that the wait for the lock succeeded. Something like:

`bool messagedReceived = lockEvent.WaitOne(timeoutReceive);
 Check.That(messagedReceived).IsTrue();`